sematic/api/endpoints/runs.py

```diff
@@ -219,6 +221,35 @@ def schedule_run_endpoint(user: Optional[User], run_id: str) -> flask.Response:
     return flask.jsonify(payload)
 
 
+@sematic_api.route("/api/v1/runs/<run_id>/logs", methods=["GET", "PUT"])
```

Comment:
Why the two methods?

Reply:
Because it's natural to think of this as a "GET" but that won't allow a json payload. The alternative is that doing a `GET` on this endpoint will give you back our application's default static content. Feels like giving the logs from the start is a more reasonable behavior.

Reply:
Yes but why not feed the three parameters of the payload as URL parameters of a GET request?
That is pretty standard in search requests. Also enables the query to be cached more easily (e.g. into Memcached/Redis after job completion), also bookmarked (although not so relevant here).

Reply:
It seemed inconsistent with our other APIs, and if we ever want to have complex parameters for this, request parameters can get awkward. But I see that the runs list is already using request parameters so I'll do that too for consistency on our APIs for reading filtered stuff in the runs API.